app/views/record/_sidebar.html.erb

```diff
@@ -8,11 +8,5 @@
     </div>
   <% end %>
 
-  <div class="bit askus">
-    <h3 class="title">Ask Us</h3>
-    <p>Not finding what you're looking for?  We're here to help with anything from quick questions to in-depth research.</p>
-    <p>
-      <a class="btn button-secondary" href="https://libraries.mit.edu/ask/">Ask Us</a>
-    </p>
-  </div>
+  <%= render 'shared/ask_gis' %>
```

Comment:
Can we rename this partial to `shared/ask` now that it is handling both use cases?

Reply:
D'oh. Yes.